src/memory_tracker.h

```diff
@@ -60,9 +67,12 @@ class NodeBIO;
  *       return "ExampleRetainer";
  *     }
  *
- *     // Or use SET_SELF_SIZE(ExampleRetainer)
+ *     // Classes that only want to return its sizeof() value can use the
+ *     // SET_SELF_SIZE(Class) macro instead.
  *     size_t SelfSize() const override {
- *       return sizeof(ExampleRetainer);
+ *       // We need to exclude the size of non_pointer_retainer so that
+ *       // we can track it separately in ExampleRetainer::MemoryInfo().
+ *       return sizeof(ExampleRetainer) - sizeof(NonPointerRetainerClass);
```

Comment:
How about `sizeof(*this)` and `sizeof(non_pointer_retainer)`?

Reply:
Ping @joyeecheung

Reply:
@TimothyGu For the example itself there probably is no difference - whatever makes the calculation easier to understand for the writer is good enough.